### ISSUE TITLE
invalid option: --without-parallel

### ISSUE THREAD

**Issue by MrCee** [slhck/homebrew-moreutils#4]
Is it just me on a new install of Big Sur? I keep getting invalid option after `brew install slhck/moreutils/moreutils --without-parallel`

**Comment by slhck** [comment]
It works for me (not Big Sur, but it shouldn't make a difference).

Make sure you have fixed all issues with `brew doctor`, possibly try uninstalling moreutils if it's already installed from the default Homebrew tap.

**Comment by MrCee** [comment]
I definitely always run `brew doctor` and sort out any permissions, all looks good.

```
➜  ~ brew uninstall moreutils
Error: No available formula or cask with the name "moreutils".
```

I'll keep digging but usually there is not much I've forgotten on a new install because I use the same script every time and have definitely updated all the commands as far as casks go, so no issues there. Everything ran perfectly except this one command.

**Comment by MrCee** [comment]
just following on before, if I install `brew install slhck/moreutils/moreutils` and let it fail because parallel has previously been installed, I can then run the command `brew install slhck/moreutils/moreutils --without-parallel`

**Comment by slhck** [comment]
"Invalid option" makes it appear as though it's using a core formula. You can also try running:

```
brew edit slhck/moreutils/moreutils
```

This should open an editor and show the formula with its options.

Other than that I'm out of ideas, sorry.

**Comment by slhck** [comment]
> just following on before, if I install `brew install slhck/moreutils/moreutils` and let it fail because parallel has previously been installed, I can then run the command `brew install slhck/moreutils/moreutils --without-parallel`

Interesting. This seems to be a bug in core `brew` behavior though; it should allow users to install with options already on the first try.

**Comment by MrCee** [comment]
Yeah a lot has changed... thank you @slhck, definitly a much needed package --without-parallel much appreciated.

**Comment by victorkristof** [comment]
Even though it doesn't make use of your tap, this solution enabled me to install `moreutils` and `GNU parallel` (I'm on Monterey):

```
brew install moreutils
brew unlink moreutils
brew install parallel
brew link --overwrite moreutils
brew unlink parallel
brew link --overwrite parallel
```

Source: https://superuser.com/a/1537888
